assets/css/_view_header.scss

```diff
@@ -1,5 +1,7 @@
 .m-view-header {
-  position: relative;
+  position: sticky;
+  top: 0;
+  z-index: 1;
```

Comment:
z-index: 1 here keeps the content from scrolling over the header. I'm sorting through why that works and if there could be a cleaner solution without specifying z-index

Reply:
update: After a little more playing around, this seems to work because of the stack context. Looking only at the notifications drawer, I was able to get the sticky header without using z-index if alerting some of the `position: relative` and `position: absolute` configurations of elements within the sibling to this header. It seems like that would result in more significant changes & open up more surface area for possible regressions, so I'm opting to leave as-is for feedback.

Reply:
If the only thing the z-index is doing here is triggering a new stacking context, could you explicitly set `isolation: isolate;`? I just learned about `isolation` from poking around MDN but it looks like it might do what we need: https://developer.mozilla.org/en-US/docs/Web/CSS/isolation

Reply:
`isolation: isolate` doesn't seem to work in this case. I may be wrong here (stacking contexts is a new concept to me), but it  seems like the behavior of the current change in this PR is that setting z-index moves the view header up in the stacking context so that it overlays any element where `position` is set and `z-index` is not. 

That is why alternatively removing `position: relative` on other elements has the same effect: the current approach of the PR moves the header up a level in the stacking context, the alternative moves everything else down a level.

Reply:
Yeah I'll confess I'm not great with stacking contexts.

In any event, I'd say it's preferable to go with the z-index solution here over the other one that involves changing the code in more places and introducing possible regressions.